### PR TITLE
Bug 1865927: Fix import wizard

### PR DESCRIPTION
**Pull request by yaacov**
Import wizard show an error when open

Screenshot:
screenshot-localhost_9000-2020 08 04-17_35_54

### CODE REVIEW

File: frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/state-update/storage-tab-state-update.ts [hunk 1]
```diff
@@ -51,17 +51,10 @@ export const prefillInitialDiskUpdater = ({ id, prevState, dispatch, getState }:
...
-    hasVMSettingsValueChanged(
-      prevState,
-      state,
-      id,
-      VMSettingsField.OPERATING_SYSTEM,
-      VMSettingsField.FLAVOR,
-      VMSettingsField.WORKLOAD_PROFILE,
-      VMSettingsField.CLONE_COMMON_BASE_DISK_IMAGE,
-    ) &&
     newSourceStorage?.dataVolume?.spec?.source?.pvc?.name !==
```
Comment (suomiy) [inline frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/state-update/storage-tab-state-update.ts:54]: +1, although it feels a bit weird not comparing the type first

File: frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/state-update/vm-settings-tab-state-update.ts [hunk 2]
```diff
@@ -176,7 +176,7 @@ const baseImageUpdater = ({ id, prevState, dispatch, getState }: UpdateOptions)
   if (!hasVMSettingsValueChanged(prevState, state, id, VMSettingsField.OPERATING_SYSTEM)) {
     return;
   }
-  if (iGetLoadedCommonData(state, id, VMWizardProps.isProviderImport)) {
```
Comment (suomiy) [inline frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/state-update/vm-settings-tab-state-update.ts:179]: can we first ask if `isProviderImport` to be consistent?

File: frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/state-update/vm-settings-tab-state-update.ts [hunk 3]
```diff
@@ -153,7 +153,7 @@ const osUpdater = ({ id, prevState, dispatch, getState }: UpdateOptions) => {
...
```
Comment (suomiy) [inline frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/state-update/vm-settings-tab-state-update.ts:156]: same here

File: frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/state-update/vm-settings-tab-state-update.ts (hunk 2 above)
Comment (yaacov) [reply frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/state-update/vm-settings-tab-state-update.ts:179]: moved up :-)

File: frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/state-update/vm-settings-tab-state-update.ts (hunk 3 above)
Comment (yaacov) [reply frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/state-update/vm-settings-tab-state-update.ts:156]: mved up

**Pull request merged**